react/features/base/util/uri.js

```diff
@@ -6,6 +6,8 @@
  */
 export const APP_LINK_SCHEME = 'org.jitsi.meet:';
 
+const _ROOM_EXCLUDE_PATTERN = '[\\:\\?#\\[\\]@!$&\'()*+,;=]';
```

Comment:
@zbettenbuk, we require jsdocs on everything (which is in our coding style and I remember telling it to you in a previous PR review)!

Reply:
Yeah, sorry, added